exporter/azuredataexplorerexporter/README.md

```diff
@@ -0,0 +1,178 @@
+# Azure Data Explorer Exporter
+
+| Status                   |           |
+| ------------------------ | --------- |
+| Stability                | [alpha]   |
```

Comment:
You'll need to add the links to `alpha` and `contrib`, see https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/d444f478e7478fdc31c46533136a9ba6bd9f433e/exporter/coralogixexporter/README.md?plain=1#L57-L59